aten/src/ATen/native/transformers/cuda/attention_backward.cu

```diff
@@ -261,7 +312,7 @@ std::tuple<at::Tensor, at::Tensor, at::Tensor> _efficient_attention_backward(
     ASSIGN_CHECK_OVERFLOW(p.gQ_strideH, grad_q.stride(2));
     ASSIGN_CHECK_OVERFLOW(p.gK_strideH, grad_k.stride(2));
     ASSIGN_CHECK_OVERFLOW(p.gV_strideH, grad_v.stride(2));
-    p.gQKV_strideM_multiplier = gQKV_strideM_multiplier;
+    p.gQKV_strideM_multiplier = grad_q.is_contiguous() ? 1 : 3;
```

Comment:
This should alwasy be 1 since I removed the chunk_grad optimization, not sure why tests didn't catch this :?

Reply:
actually tests pass since grad_q is always contiguous,  make this explicit